src/rpc/net.cpp

```diff
@@ -119,7 +119,7 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
                             {
                                 {RPCResult::Type::NUM, "n", "The heights of blocks we're currently asking from this peer"},
                             }},
-                            {RPCResult::Type::BOOL, "whitelisted", "Whether the peer is whitelisted"},
+                            {RPCResult::Type::BOOL, "whitelisted", /* optional */ true, "Whether the peer is whitelisted with default permissions (DEPRECATED, returned only if config option -deprecatedrpc=whitelisted is passed)"},
```

Comment:
This line is much longer than any of the other ones. Consider newlining to display it in two lines (perhaps do the same with the help at line 134 while here).

Reply:
![Screenshot from 2020-08-27 10-41-47](https://user-images.githubusercontent.com/2415484/91418292-7879fc00-e841-11ea-8067-f70f257c2ed1.jpg)

Reply:
Isn't formatting the job of RPCHelpMan?

Reply:
RPCHelpMan doesn't (yet) add line breaks.

Regardless, this string will go away on November 1st, so no need to over-optimize something that will be removed.